components/sysutils/open-vm-tools/open-vm-tools.p5m

```diff
@@ -65,6 +65,7 @@ file path=usr/lib/$(MACH64)/libhgfs.so.0.0.0
 link path=usr/lib/$(MACH64)/libvmtools.so target=libvmtools.so.0.0.0
 link path=usr/lib/$(MACH64)/libvmtools.so.0 target=libvmtools.so.0.0.0
 file path=usr/lib/$(MACH64)/libvmtools.so.0.0.0
+link path=usr/lib/$(MACH64)/open-vm-tools/plugins/$(MACH64) target=.
```

Comment:
Should we do it? This looks ugly.
Better patch open-vm-tools-stable-10.3.0/services/vmtoolsd/pluginMgr.c:
It has 
```
#if defined(sun) && defined(__x86_64__)
   const char *subdir = "/amd64";
#else
   const char *subdir = "";
#endif
```
Just set it to ""

Reply:
Thanks for finding the culprit. I removed the uglyness and re-tested.

Reply:
Sorry, I don't see the patch in PR.

Reply:
Thanks for catching it, forgot to add it.